src/menu.rs

```diff
@@ -155,15 +161,21 @@ pub fn run(mut window: &mut PistonWindow,
                             match menu_selection {
                                 MenuSelection::Play => {
                                     music::play(&Music::Action, music::Repeat::Forever);
-                                    game::Game::new(window_size)
-                                        .run(&mut window, &mut opengl, &mut glyph_cache);
+                                    game::Game::new(window_size).run(&mut events,
+                                                                     &mut window,
+                                                                     &mut opengl,
+                                                                     &mut glyph_cache);
                                     music::play(&Music::Menu, music::Repeat::Forever);
                                 }
                                 MenuSelection::Story => {
-                                    story::run(&mut window, &mut opengl, &mut glyph_cache);
+                                    story::run(&mut events,
```

Comment:
Ah, cool. Well I think what this endevour has proven, which we didn't know before, was that `piston_window` really is giving us a bit of convenient functionality over piston core and I think it would be clearer to use it to manage the piston dependencies and this feature. Unless you had any other ideas, I'm leaning towards closing #72 and referencing this work as reasoning why.

Again, thanks for the work you've done!